### PR TITLE
PYTHON: Add helper methods to be able to easily create all the geometries.

### PR DESCRIPTION
**Pull request by joeykleingers**
Adds the following helper methods to be able to easily create all the geometries:

nx.CreateGeometry.create_image_geometry
nx.CreateGeometry.create_rect_grid_geometry
nx.CreateGeometry.create_vertex_geometry
nx.CreateGeometry.create_edge_geometry
nx.CreateGeometry.create_triangle_geometry
nx.CreateGeometry.create_quad_geometry
nx.CreateGeometry.create_tetrahedral_geometry
nx.CreateGeometry.create_hexahedral_geometry

Also adds parameter names to the ChoicesParameter binding.

### CODE REVIEW

**Comment by imikejackson** [comment]
We talked about the API returning a Result<> object.

**Comment by joeykleingers** [comment]
> We talked about the API returning a Result<> object.

They all return Result objects.

**Review by imikejackson — APPROVED** [review]

**Comment by JDuffeyBQ** [comment]
Were we going to merge the various `ArrayHandlingType` enums/values for these bindings?

**Comment by imikejackson** [comment]
@JDuffeyBQ Those changes should be on https://github.com/BlueQuartzSoftware/simplnx/pull/983/

**Comment by JDuffeyBQ** [comment]
Ah then that enum should be bound using `py::enum_` here.

File: src/Plugins/SimplnxCore/wrapping/python/simplnxpy.cpp [hunk 1]
```diff
@@ -1520,4 +1523,170 @@ PYBIND11_MODULE(simplnx, mod)
   manualImportFinder.def("clear", &ManualImportFinder::clear);
   manualImportFinder.def("contains_path", &ManualImportFinder::containsPath, "path"_a);
   manualImportFinder.def("contains_module", &ManualImportFinder::containsModule, "mod_name"_a);
+
+  // Geometry Helper Methods
+  py::module_ sub = mod.def_submodule("ArrayHandlingType", "How existing arrays will be handled when creating the new geometry.");
+  sub.attr("CopyArray") = CreateGeometryFilter::k_CopyArray;
+  sub.attr("MoveArray") = CreateGeometryFilter::k_MoveArray;
```
Comment (JDuffeyBQ) [inline src/Plugins/SimplnxCore/wrapping/python/simplnxpy.cpp]: Can be replaced by the binding made in https://github.com/BlueQuartzSoftware/simplnx/pull/983 though it needs to be updated to include all values. And the functions after this should be updated as well.

**Review by JDuffeyBQ — CHANGES_REQUESTED** [review]

File: src/Plugins/SimplnxCore/wrapping/python/simplnxpy.cpp (hunk 1 above)
Comment (imikejackson) [reply src/Plugins/SimplnxCore/wrapping/python/simplnxpy.cpp]: @joeykleingers Can you take care of this? If so we could get everything we need for a release.

**Comment by joeykleingers** [comment]
This is ready for review again.

**Review by imikejackson — APPROVED** [review]

**Pull request merged**